aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AutoEventSubmitter.java

```diff
@@ -41,8 +39,11 @@ final class AutoEventSubmitter {
         this.handler = new Handler(handlerThread.getLooper());
```

Comment:
Should we remove `analyticsClient` from the constructor since APIs are directly accessed using plugin now?